frontend/src/editor/transcription_editor.tsx

```diff
@@ -123,20 +150,21 @@ export function TranscriptionEditor({
             /* the value is actually managed by the editor object */
           ]
         }
-        onChange={() => {
+        onChange={useCallback(() => {
           // set the confidence of manually edited nodes to 1.0
           const hasChanged = editor.operations.some(
-            (op) => op.type == 'insert_text' || op.type == 'remove_text',
+            (op: Operation) => op.type == 'insert_text' || op.type == 'remove_text',
           );
           if (hasChanged) {
             Transforms.setNodes(editor, { conf: 1.0 }, { match: (n) => 'conf' in n });
           }
-        }}
+        }, [])}
       >
         <Editable
-          renderElement={(props) => renderElement(props)}
+          renderElement={useCallback((props: RenderElementProps) => renderElement(props), [])}
```

Comment:
see above

Reply:
This one definitely help a lot, because otherwise the `renderElement` function changes every time `TranscriptionEditor` is rendered, which in turn causes all of the elements to be rerendered.

Reply:
if we dont use a lambda here, function equality should hold as-is